stepup/gateway/docker-compose.override.yml

```diff
@@ -1,5 +1,7 @@
 services:
   gateway:
-    image: ghcr.io/openconext/stepup-gateway/stepup-gateway:dev
+    image: ghcr.io/openconext/openconext-basecontainers/${GATEWAY_PHP_IMAGE:-php72-apache2-node14-composer2:latest}
     volumes:
       - ${GATEWAY_CODE_PATH}:/var/www/html
+    environment:
+      - APP_ENV=dev
```

Comment:
@MKodde does we have to add ` APP_DEBUG: true` to that?

Reply:
@quartje Pauls suggestion seems to make sense to me. Why not enable the debugger while we're at it?

Reply:
sure. I've added it!